src/node_crypto.cc

```diff
@@ -140,6 +140,13 @@ static const char* const root_certs[] = {
 #include "node_root_certs.h"  // NOLINT(build/include_order)
 };
 
+static const char* system_cert_path =
+#if defined(NODE_OPENSSL_SYSTEM_CERT_PATH)
+        NODE_STRINGIFY(NODE_OPENSSL_SYSTEM_CERT_PATH);
+#else
+        nullptr;
+#endif
```

Comment:
`static const char system_cert_path[] = ...;` (`""` when undefined) and I'd prefer defining it in both arms, easier to read.

`NODE_STRINGIFY()` isn't necessary, or is it?

Reply:
I'll define the in both cases and use "" when undefined. 

I do think the NODE_STRINGIFY call is needed to get the value (the path) but if there are better ways of doing this please let me know. Thanks